### PR TITLE
release-schema: Update amendments field descriptions

### LINKED ISSUE #1154: Ensure tender field descriptions make sense for planning and contracting processes
Follow-up to https://github.com/open-contracting/standard/issues/866#issuecomment-751683275

> @jpmckinney: Publishers can create OCDS releases/records for planning processes, even though these are not the same as contracting processes. The contracting process(es) that result from the planning process should refer to it using `relatedProcesses` and use different OCID(s). Those contracting processes should omit the `planning` section. As for the planning processes, they should omit the `awards` and `contracts` section. Right now, in OCDS, the `tender` section covers fields relevant to describing the procedure (not just the solicitation), so planning processes might include some `tender` fields.

> @JachymHercher: I'm wondering what implications this ambiguity (`Tender` block being used both during the "planning stage" and "tender stage") has for definitions of `Tender`, `Planning` and potentially also others blocks. Perhaps their definitions should not refer to "stages" at all, but simply talk about the data that they contain.

### PR DESCRIPTION
**Pull request by duncandewhurst**
* Closes #1154 and makes equivalent changes to `awards.amendments` and `contracts.amendments`

### CODE REVIEW

File: schema/release-schema.json [hunk 1]
```diff
@@ -525,7 +525,7 @@
           }
         },
         "amendments": {
-          "description": "A tender amendment is a formal change to the tender, and generally involves the publication of a new tender notice/release. The rationale and a description of the changes made can be provided here.",
+          "description": "Formal changes to the tender, which generally involve the publication of a new notice/release.",
```
Comment (odscjen) [inline schema/release-schema.json]:
```suggestion
          "description": "The formal changes to the tender, which generally involve the publication of a new notice/release.",
```

more in keeping with the style guide to start with an [article](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#articles)

File: schema/release-schema.json [hunk 2]
```diff
@@ -716,7 +716,7 @@
           "uniqueItems": true
         },
         "amendments": {
-          "description": "An award amendment is a formal change to the details of the award, and generally involves the publication of a new award notice/release. The rationale and a description of the changes made can be provided here.",
+          "description": "Formal changes to the award, which generally involve the publication of a new notice/release.",
```
Comment (odscjen) [inline schema/release-schema.json]:
```suggestion
          "description": "The formal changes to the award, which generally involve the publication of a new notice/release.",
```

File: schema/release-schema.json [hunk 3]
```diff
@@ -901,7 +901,7 @@
           }
         },
         "amendments": {
-          "description": "A contract amendment is a formal change to, or extension of, a contract, and generally involves the publication of a new contract signature notice/release, or some other documents detailing the change. The rationale and a description of the changes made can be provided here.",
+          "description": "Formal changes or extensions to the contract, which generally involve the publication of a new notice/release.",
```
Comment (odscjen) [inline schema/release-schema.json]:
```suggestion
          "description": "The formal changes or extensions to the contract, which generally involve the publication of a new notice/release.",
```

**Review by odscjen — CHANGES_REQUESTED** [review]
minor copy edits

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**